script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -261,71 +267,72 @@ But for now we should hurry to school. You're not gonna be late on my watch."
+All the other students that had been milling around by the entrance disappeared while I wasn't paying attention.
+ I spent so long trying to get an answer out of Senpai that there's now only three minutes left until homeroom.
 }
 [sha:93bb88db7099ca1716b7cddb81aab0205a70834b]{
 -- Page 54, Offset 14933.
 -- 文句もあるけど今は先輩の言う通りだ。
 // I added the latter sentence, maybe it's not needed.
-I still have things I want to say to her, but she's right: I have to go to class.
+I'm still not done with her, but she's right――I have to get to class.
 }
 [sha:9904d2924cc5eb600e9068ad9c20c717c255f264]{
 -- Page 54, Offset 14934.
 -- まだ動転している心臓を落ち着かせながら、先輩の後に付いていった。
-I try to calm my palpitating heart as I follow Senpai toward the building.
+Trying to calm my racing heart, I follow after Senpai.
 }
 [sha:7533c56d731d9fd7e38b02635b72114e2e2d8215]{
 -- Page 55, Offset 14935.
 -- ホームルームまであと１分。
-There's 1 minute left until homeroom starts.
+There's only one minute left before homeroom starts.
 }
 [sha:974e285f4570ef3b8cc7a2008fab4cdb3b918d3f]{
 -- Page 55, Offset 14936.
 -- 廊下に生徒の姿はない。
-There are no students in the corridors.
+The corridor is completely devoid of students.
 }
 [sha:cdb0167e400b00b0cd6723a84d3d172865a2b7c9]{
 -- Page 55, Offset 14937.
 -- あるのは遅刻間際で小走りになる俺と先輩の足音と、
 // found this hard to phrase; pay special attention in editing
-The only thing there is the sound of footsteps from the two people almost late for class.
+The only things present in it are the footsteps of two people that are on the verge of being late for class, and―――
 }
 [sha:be3ade05ea3b04e55e983566b634e889ae6e8789]{
 -- Page 56, Offset 14938.
 -- 「あ、きたきた～♪　志貴クン、おはよー！」
-"Oh, arrived at last ♪
- Mornin' Shiki-kun!"
+"Oh, here he comes ♪%{n}
+Mornin' Shiki-kun!"
 }
 [sha:1fd29d6120e0c33609f880cceb4d3df1cb992d4b]{
 -- Page 56, Offset 14939.
 -- 踊り場で待ち受けていた、ノエル先生の笑顔だった。
-A smiling Noel-sensei greets me in the middle of the stairs.
+A smiling Noel-sensei greets me from the landing.
 }
 [sha:892f9d27dae06b1951da763b7882372e583e4671]{
 -- Page 57, Offset 14940.
 -- 「ノエル先生!?」
-"Noel-sensei!?"
+"Noel-sensei?!"
 }
 [sha:b8b5c023f147c913a68e33d243d73052db8db550]{
 -- Page 57, Offset 14941.
 -- 声をあげたのは先輩の方だ。
-Surprisingly, Senpai is the one who exclaims as much.
+It's actually Senpai who calls out in surprise.
 }
 [sha:bdc7b4dab9f841c4e97ccd669b95ebffb4ec6ad9]{
 -- Page 57, Offset 14942.
 -- 俺はというと、さっきの先輩のサプライズが大きすぎて、ノエル先生がいても“先輩がいるんだから、ノエル先生だっているだろう”ぐらいの感想しか浮かばなかった。
-As for me, my only reaction is thinking that of course Noel-sensei would be around if Senpai still is.
- I suppose I was in such shock at seeing Senpai, that Noel-sensei still being around doesn't elicit much of a reaction.
+I'm still reeling from the shock of running into Senpai earlier, so seeing Noel-sensei here doesn't elicit much of a reaction from me.
+ Besides, I'd figured that if Senpai is still here, it stands to reason that Noel-sensei would be too.
```

Comment:
"I'd figured" mismatched with "it stands" 
"I'd figured [that] if Senpai is still here, it'd stand to reason that Noel-sensei would be too."